creme/utils/skmultiflow_utils.py

```diff
@@ -237,3 +238,28 @@ def add_delay_to_timestamps(timestamps, delay):
     for t in timestamps:
         delay_timestamps.append(t + delay)
     return np.array(delay_timestamps, dtype="datetime64")
+
+
+def check_random_state(seed):
```

Comment:
This function is also included in the generators' PR. This will raise some issues when merging. Just FYI.

Reply:
Good catch. I will handle possible conflicts when pulling the upstream.